.github/workflows/docs.yml

```diff
@@ -26,6 +33,9 @@ jobs:
       - name: Build
         run: cd docs && npm install && hugo --minify
 
+      - name: ls
+        run: ls ./docs/public
```

Comment:
why we need that?

Reply:
That was just to [get a little debugging output](https://github.com/kubernetes-sigs/bom/pull/358/commits/4668d9d783b83a0da16d0bd0130712cc6fa85763). It and the pull_request commit would be dropped before we would want to merge this.